### PR TITLE
[Outlook] (SSO) Fix quick start instructions

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Fixes https://github.com/OfficeDev/office-js-docs-pr/issues/2260.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1732ec4](https://github.com/OfficeDev/office-js-docs-pr/commits/1732ec4659cb5b10e4e492387d8ad22213e74ec6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstarts/sso-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-fix-2260/docs/quickstarts/sso-quickstart.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/sso-quickstart?branch=pr-en-us-2304) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202011031746553450-2304%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=sLxjr09j4etOWfWg6owh5BmnX2t8C6rn%2BTebKb40TWQ%3D&st=2020-11-03T17%3A42%3A35Z&se=2020-12-04T17%3A47%3A35Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/quickstarts/sso-quickstart.md [hunk 1]
```diff
@@ -119,24 +119,24 @@ Complete the following steps to try out an Excel, Word, or PowerPoint add-in.
 
 Complete the following steps to try out an Outlook add-in.
 
-1. When the SSO configuration process completes, run the following command to build the project and start the local web server.
+1. When the SSO configuration process completes, run the following command to build the project and start the local web server, and sideload your add-in in the previously selected Office client application.
```
Comment (davidchesnut) [inline docs/quickstarts/sso-quickstart.md:122]:
```suggestion
1. When the SSO configuration process completes, run the following command to build the project and start the local web server, and sideload your add-in in Outlook on Windows.
```

File: docs/quickstarts/sso-quickstart.md [hunk 2]
```diff
@@ -119,24 +119,24 @@ Complete the following steps to try out an Excel, Word, or PowerPoint add-in.
...
 
     > [!NOTE]
     > Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides. You may also have to run your command prompt or terminal as an administrator for the changes to be made.
 
     ```command&nbsp;line
-    npm run dev-server
+    npm start
```
Comment (davidchesnut) [inline docs/quickstarts/sso-quickstart.md:128]: This won't launch Outlook and sideload. You still need to link to those instructions.

File: docs/quickstarts/sso-quickstart.md [hunk 3]
```diff
@@ -119,24 +119,24 @@ Complete the following steps to try out an Excel, Word, or PowerPoint add-in.
...
     ```
 
-2. Follow the instructions in [Sideload Outlook add-ins for testing](../outlook/sideload-outlook-add-ins-for-testing.md) to sideload the add-in in Outlook. Make sure that you're signed in to Outlook with a user that's a member of the same Microsoft 365 organization as the Microsoft 365 administrator account that you used to connect to Azure while configuring SSO in step 3 of the [previous section](#configure-sso). Doing so establishes the appropriate conditions for SSO to succeed.
+2. Make sure that you're signed in to Outlook with a user that's a member of the same Microsoft 365 organization as the Microsoft 365 administrator account that you used to connect to Azure while configuring SSO in step 3 of the [previous section](#configure-sso). Doing so establishes the appropriate conditions for SSO to succeed.
```
Comment (davidchesnut) [inline docs/quickstarts/sso-quickstart.md:131]: You'll still need this link for sideloading.

**Review by davidchesnut — CHANGES_REQUESTED** [review]
You still need to include sideloading instructions...

File: docs/quickstarts/sso-quickstart.md (hunk 1 above)
Comment (TCourtneyOwen) [reply docs/quickstarts/sso-quickstart.md:122]: There is no sideloading for Outlook yet, so the above string should be updated to not include "sideload" and we should provide the instructions for how a user manually registers an add-in for Outlook

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ed3a0f5](https://github.com/OfficeDev/office-js-docs-pr/commits/ed3a0f596e4071c674b7a8df9e73cdf693ddf4d6)_: 

:clock10: **Pending**: waiting for processors ([56](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 56 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by ElizabethSamuel-MSFT** [comment]
@davidchesnut @TCourtneyOwen Thanks for the feedback. Update to keep the sideloading instructions.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ed3a0f5](https://github.com/OfficeDev/office-js-docs-pr/commits/ed3a0f596e4071c674b7a8df9e73cdf693ddf4d6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstarts/sso-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-fix-2260/docs/quickstarts/sso-quickstart.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/sso-quickstart?branch=pr-en-us-2304) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202011031845547768-2304%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=rPGFDMo7vr0V5GyuKoTs%2BSjqJEMt9iIiNtmvfY6SojM%3D&st=2020-11-03T18%3A54%3A16Z&se=2020-12-04T18%3A59%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/quickstarts/sso-quickstart.md (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/quickstarts/sso-quickstart.md:128]: Kept sideloading instructions

**Review by TCourtneyOwen — APPROVED** [review]

**Pull request merged**